numpy/linalg/setup.py

```diff
@@ -41,7 +41,10 @@ def calc_info(self):
                 info = {'language': 'c'}
                 if sys.maxsize > 2**32:
                     # Build lapack-lite in 64-bit integer mode
-                    info['define_macros'] = [('HAVE_BLAS_ILP64', None)]
+                    info['define_macros'] = [
+                        ('HAVE_BLAS_ILP64', None),
+                        ('BLAS_SYMBOL_SUFFIX', '64_')
```

Comment:
Is this chosen arbitrary, or does it match something else? If that latter, can we add a comment here?

Reply:
It's arbitrary. Added comment.